### ISSUE TITLE
question: Why does `tfsec` ask for Github creds

### ISSUE THREAD

**Issue by lemonbakeoff** [aquasecurity/tfsec#1716]
Tried to build and run `tfsec` from within Docker container.

Now, it's asking...
`Username for 'https://github.com': `

Why???

**Comment by lemonbakeoff** [comment]
Looks like `tfsec` is reaching out to clone modules.
Using `--no-module-downloads` flag is workaround.

If modules are in public repo, then I don't see why providing credentials should be necessary?

**Comment by owenrumney** [comment]
Hey @lemonbakeoff  - tfsec isn't asking for the creds, its likely that go-getter is asking for them, I assume you get a similar thing when doing a terraform init as were replicating the approach they use.

As to why its asking for public repos, without seeing the `source` attribute I couldn't say, I wonder are you using `git@github.com:path/tomodule`?

**Comment by lemonbakeoff** [comment]
Thanks @owenrumney.

I believer I am using `git@github.com:path/tomodule`.
It's the default inside an Alpine container for `apk add git`.

In the past, I've had to modify global URL for git inside Alpine containers...
`git config --global url."https://github.com/".insteadOf git://github.com/`

However, I tried that here and had the same issue.

Without confirming, I'm going to assume that you're right about `go-getter`.
Therefore, if I want `tfsec` to scan modules as well, I'll need to come up with a workaround (e.g. passing Github creds to container). Definitely doable, but not for an initial proof of concept.

Closing, thanks.
